src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -338,69 +340,55 @@ namespace OpenMS
 
   Size ProteinIdentification::computeCoverage(const std::vector<PeptideIdentification>& pep_ids)
```

Comment:
How should `computeCoverage` behave when there is no peptide evidence for a certain protein?
(1) Exception, because `ProteinIdentification` run and peptide identifications `pep_ids` are not consistent.
(2) Return `coverage = 0` for these proteins.
IMO, the latter.